.gitignore

```diff
@@ -28,4 +28,5 @@ custom-elements.json
 *.sln
 *.sw?
 .nx/*
-agentic.stories.js
\ No newline at end of file
+agentic.stories.js
+/.history
```

Comment:
where is this `history` folder coming from?

Reply:
they're from my VSCode settings; I'll remove the line from the .gitignore